LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2023 Buf Technologies, Inc.
```

Comment:
Per the instructions above I'm not sure this file is meant to have the actual copyright holder written in. Of course instructions can be changed.

Reply:
Yeah, the Buf license situation is to set the value